nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -657,6 +657,11 @@ systemd.services.nginx.serviceConfig.ReadWritePaths = [ "/var/www" ];
      since Nextcloud doesn't support upgrades across multiple major versions.
     </para>
    </listitem>
+   <listitem>
+    <para>
+      The packages <package>perl</package>, <package>rsync</package> and <package>strace</package> where removed from <option>systemPackages</option>. If you need them, install them again with <code><xref linkend="opt-environment.systemPackages"/> = with pkgs; [ perl rsync strace ];</code> in your <filename>configuration.nix</filename>.
```

Comment:
```suggestion
      The packages <package>perl</package>, <package>rsync</package> and <package>strace</package> were removed from <option>systemPackages</option>. If you need them, install them again with <code><xref linkend="opt-environment.systemPackages"/> = with pkgs; [ perl rsync strace ];</code> in your <filename>configuration.nix</filename>.
```

Reply:
Should it say `<option>environment.systemPackages</option>` rather than `<option>systemPackages</option>`? I would suggest the change formally, but I'm not familiar enough with NixOS release notes norms.

Reply:
IMHO it is well known option and also can be deduced from the context but we could use `<option xlink:href="#opt-environment.systemPackages">systemPackages</option>` if we really make it unambiguous.